v2/yarpchttp/outbound.go

```diff
@@ -344,11 +331,12 @@ func getYARPCErrorFromResponse(httpRes *http.Response, resBuf *yarpc.Buffer, bot
 			code = errorCode
 		}
 	}
-	return yarpcerror.New(
-		code,
-		strings.TrimSuffix(contents, "\n"),
-		yarpcerror.WithName(httpRes.Header.Get(ErrorNameHeader)),
-	)
+	message := httpRes.Header.Get(ErrorMessageHeader)
+	return yarpcerror.Info{
+		Code:    code,
+		Message: strings.TrimSuffix(message, "\n"),
+		Name:    httpRes.Header.Get(ErrorNameHeader),
+	}
 }
```

Comment:
I realize that this was a decision we accepted, but we should note that this is technically wire-breaking, i.e. we are always returning the error along the header as opposed to sometimes along the response `yarpc.Buffer`.